optimum/commands/onnxruntime/optimize.py

```diff
@@ -0,0 +1,71 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from ...onnxruntime.configuration import AutoOptimizationConfig
+from ...onnxruntime.optimization import ORTOptimizer
+
+
+def parse_args_onnxruntime_optimize(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "--onnx_model",
```

Comment:
Why not make that just a regular argument:
```suggestion
        "onnx_model",
```

That way we can do:

```bash
optimum-cli onnxruntime optimize path_to_my_model -O2 my_output
```

Which seems less heavy IMO.

@fxmarty and maybe we should do the same for exporters, wdty?

Reply:
Yes we can do that for the `onnx_model` argument.

Reply:
I think having several unnamed arguments makes things less readable. I'm not in favor personally, but it's taste.